packages/mds-audit/tests/api.spec.ts

```diff
@@ -546,7 +546,8 @@ describe('Testing API', () => {
       devices_c = makeDevices(10, now(), MOCHA_PROVIDER_ID)
 
       const seedData = {
-        devices: [...devices_a, ...devices_b, ...devices_c],
+        // Include a duplicate device (same vin + provider but different device_id)
+        devices: [...devices_a, ...devices_b, ...devices_c, { ...devices_c[0], ...{ device_id: uuid() } }],
         events: [...events_a, ...events_b],
         telemetry: [...telemetry_a, ...telemetry_b]
```

Comment:
Are the `...` necessary here?

Reply:
I think so, I get a `SyntaxError` without them. Not sure if there's a cleaner way to do this.

Reply:
Yeah, I think that this is the most concise way to do it.